src/utils/Logger.ts

```diff
@@ -163,36 +168,13 @@ class Logger {
   }
 
   getLogsToAttach = async () => {
-    const toAttach: Email['attachments'] = []
+    const toAttach: Attachments = []
     try {
       const logDir = this.getReactNativeLogsDir()
       const logFiles = await RNFS.readDir(logDir)
-      // On Android we need to move the files to a directory we have access to
```

Comment:
is this change related?

Reply:
like did we only do this to enable emailing logs..? I can't really tell

Reply:
Yes this was only done to enable emailing logs.